components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -188,6 +188,18 @@ subroutine lin_strat_chem_inti(phys_state)
 !     
 !   initialize the linoz data
 
+    !!set true for linoz_v3 interpolation in interpolation
+    if (linoz_v3) file%linoz_v3=.true.
+    if (linoz_v2) file%linoz_v2=.true.
+    !!
+    if (linoz_v2.eq..true. .and.linoz_v3.eq..true. ) then
+            write(iulog,*) "Error: Both linoz_v2 and linoz_v3 are true, please check"
```

Comment:
Lines 196 and 199, it may not be "an error" for configurations that don't use Linoz.

Additionally, I'd recommend you print out only once at the beginning of the simulation. The current implementation is for every timestep. You may use the Linoz output of the following line in the atm.log as an example.
`linoz_readnl, linoz_lbl:           4`

Reply:
Should we use return for this then? Or just let it run? Please check the update.
For this place, it is in lin_strat_chem_inti. So it will only do once in initialization.